x-pack/solutions/observability/plugins/slo/server/services/delete_slo.ts

```diff
@@ -30,32 +31,36 @@ export class DeleteSLO {
     private rulesClient: RulesClientApi
   ) {}
 
-  public async execute(sloId: string): Promise<void> {
-    const slo = await this.repository.findById(sloId);
+  public async execute(params: BulkDeleteSLOParams): Promise<void> {
+    await Promise.all(
```

Comment:
I have a few concerns here:
- we could potentially timeout before finishing the bulk deletion process, on overview prod it takes about 1-2s for a single deletion to go through.
- we are going to create 2*N delete by query requests. This can be costly on the elasticsearch cluster. For example in production we have 5 billion SLI documents in the v3 index only. Some SLO have millions of documents. 😱